src/components/blog/blog.component.ts

```diff
@@ -1,16 +1,60 @@
 import { Component } from '@angular/core';
+import CardBlogComponent from './card-blog.component';
+import CodeOfConductImg from '@assets/images/code-of-conduct.webp';
+import TutorialImg from '@assets/images/tutorial.webp';
+import PromptIcon from '@assets/icons/prompt-icon.svg';
 
 @Component({
-  selector: 'app-blog',
+  selector: 'app-blog-component',
   standalone: true,
+  imports: [CardBlogComponent],
   template: `
-    <ul class="list-disc pl-5">
-      <li class="mb-2">
-        <a href="/posts/post-1/" class="text-blue-500 hover:text-blue-700"
-          >Post 1</a
+    <article class="bg-[#111828] pb-16">
+      <header class="flex flex-col items-center gap-4 py-16 px-4 md:px-8">
+        <h2
+          class="flex gap-1 md:gap-2 text-white text-4xl font-bold text-center"
         >
-      </li>
-    </ul>
+          <img [src]="PROMPT_ICON_IMG" alt="Prompt Icon" />
+
+          {{ TITLE }}
+        </h2>
+      </header>
+
+      <section
+        class="flex flex-col md:flex-row gap-8 px-4 justify-center items-center md:px-8"
+      >
+        <app-card-blog
+          [title]="CODE_OF_CONDUCT"
+          [content]="CODE_OF_CONDUCT_CONTENT"
+          [date]="CODE_OF_CONDUCT_DATE"
+          [image]="CODE_OF_CONDUCT_IMAGE"
+          [time]="CODE_OF_CONDUCT_TIME"
+          [link]="CODE_OF_CONDUCT_LINK"
+        ></app-card-blog>
+
+        <app-card-blog
```

Comment:
I know you've been told to add this, sorry for that, but don't do it, just remove this cad until we have the tutorial ready.